dolphin/db/sqlalchemy/api.py

```diff
@@ -38,6 +41,18 @@
                         connection=_DEFAULT_SQL_CONNECTION)
 
 
+def apply_sorting(model, query, sort_key, sort_ord):
```

Comment:
what's is sort_ord?

Reply:
it is sort_dir, suggestion from Ashit to to name it as (sort order)

Reply:
but othere place are using sort_dir, why not keep sort_dir?

Reply:
ok, done, keeping sort_dir as you suggested